Runtime/Scripts/PlayerData/InstanceInfoRecord.cs

```diff
@@ -0,0 +1,9 @@
+﻿using UnityEngine;
+
+namespace Unity.MeshSync
+{
+    internal class InstanceInfoRecord
+    {
+        public GameObject go;
+    }
```

Comment:
This is WIP, for the moment I wanted to mimic how instances are managed. Am I right to think that the EntityRecord related to the CachePlayer? could we use the InstanceInfoRecord as a way to store instancing info on the cache?